docs/en/writing-running-appium/server-args.md

```diff
@@ -40,20 +40,20 @@ All flags are optional, but some are required in conjunction with certain others
 |`--nodeconfig`|null|Configuration JSON file to register appium with selenium grid|`--nodeconfig /abs/path/to/nodeconfig.json`|
 |`-ra`, `--robot-address`|0.0.0.0|IP Address of robot|`--robot-address 0.0.0.0`|
 |`-rp`, `--robot-port`|-1|port for robot|`--robot-port 4242`|
-|`--chromedriver-port`|9515|Port upon which ChromeDriver will run|`--chromedriver-port 9515`|
-|`--chromedriver-executable`|null|ChromeDriver executable full path||
+|`--driver-args '{"uiautomator2": {"chromedriver-port": 9515}}`|9515|Port upon which ChromeDriver will run|`--driver-args '{"uiautomator2": {"chromedriver-port": 9515}}`|
```

Comment:
the appium 2.0 documentation will not include driver specific info like this; we should move these values to a table in the uiautomator2 driver readme!